dev/RadioButtons/RadioButtons.idl

```diff
@@ -12,27 +12,30 @@ unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
 
     Object ItemsSource;
     Windows.Foundation.Collections.IVector<Object> Items{ get; };
-    [MUX_DEFAULT_VALUE("winrt::RadioButtonsElementFactory{}")]
+    [MUX_DEFAULT_VALUE("winrt::make<RadioButtonsElementFactory>()")]
     Object ItemTemplate;
 
```

Comment:
@ranjeshj I get why this is Object but I think it will be confusing, I thought we had a way where Repeater could use either DataTemplate or ElementFactory with a common interface?

Reply:
@ranjeshj suggested making this object to support DataTemplateSelectors as well.  If we didn't care about that I think we could use IElementFactory, because DataTemplate implements that (although not down level).

Reply:
ItemsRepeater.ItemTemplate is of type object because IElementFactory cannot be used from the platform because the args are not available downlevel. I think this is the best option until WinUI3 where we can change this property type to IElementFactory.